ruby-backend-starter-code/.idea/workspace.xml

```diff
@@ -0,0 +1,744 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not an Intellij user. Should `workspace.xml` be git-ignored? https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems